<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LT-53JXL63" failures="1" tests="1" name="Today_Test" time="3.926" errors="0" timestamp="2022-03-31T10:48:30 IST">
  <testcase classname="testng.AssertionsClass" name="assetionMethod" time="3.889">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [4.1.2] but found [4.1.3]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [4.1.2] but found [4.1.3]
at testng.AssertionsClass.assetionMethod(AssertionsClass.java:23)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- assetionMethod -->
</testsuite> <!-- Today_Test -->
